src/Quadmath.jl

```diff
@@ -361,6 +363,8 @@ eps(::Type{Float128}) = reinterpret(Float128, 0x3f8f_0000_0000_0000_0000_0000_00
 floatmin(::Type{Float128}) = reinterpret(Float128, 0x0001_0000_0000_0000_0000_0000_0000_0000)
 floatmax(::Type{Float128}) = reinterpret(Float128, 0x7ffe_ffff_ffff_ffff_ffff_ffff_ffff_ffff)
 
+maxintfloat(Float128) = Float128(10384593717069655257060992658440192)
```

Comment:
Should be `maxintfloat(::Type{Float128}) = ...`

Might be prettier in hex? (`0x0002_0000_0000_0000_0000_0000_0000_0000`)

Reply:
```suggestion
maxintfloat(::Type{Float128}) = Float128(0x0002_0000_0000_0000_0000_0000_0000_0000)
```